src/d3a/models/area/__init__.py

```diff
@@ -142,6 +143,7 @@ def __init__(self, name: str = None, children: List["Area"] = None,
         self.redis_ext_conn = RedisMarketExternalConnection(self) \
             if external_connection_available and self.strategy is None else None
         self.should_update_child_strategies = False
+        self.offers_bids_matcher = MycoMatcher()
```

Comment:
Sorry to leave only renaming comments for now. But IMO a consistent naming scheme is helping understanding, also later when working with the code.
Here I would vote to be consistent and rename to:
```suggestion
        self.bid_offer_matcher = MycoMatcher()
```

Reply:
Also: There are two steps now: `recommend` and `match` . I haven't found a perfect solution for naming this yet, but really the `MycoMatcher` is really a `MycoRecommendator` (Please dont name it like this :-)) This is just a comment which you maybe remember later on. Thanks